components/log-viewer-webui/server/src/plugins/MongoSocketIoServer/index.ts

```diff
@@ -34,6 +34,9 @@ import {
 } from "./utils.js";
 
 
+// Counter for generating unique query IDs.
+let globalQueryIdCounter = 0;
```

Comment:
gg... sorry for missing this in the previous review.

can we make this variable a member of the MongoSocketIoServer singleton class?